### ISSUE TITLE
[BUG] Theme does not apply everywhere

### ISSUE THREAD

**Issue by eth0net** [massif-press/compcon#2320]
**Describe the bug**
The selected theme does not apply to the mech loadout screen under a pilot profile, or to the title bar/background of the PWA.

**To Reproduce**
1. Select a non-default theme
2. Go to the `pilot page`
3. Click on `active mech loadout`
4. See the red colours on the page

**Expected behavior**
The red is replaced with theme colours same as in all the other pages.

**Screenshots**

![Screenshot 2023-10-15 033059](https://github.com/massif-press/compcon/assets/1347854/83dac07e-c49d-4749-91a8-f4c2b1fbd447)

**Desktop:**
 - OS: Windows 11
 - Browser: Edge
 - Version: 118

**Smartphone:**
 - Device: iPhone16,2
 - OS: iOS 17.1
 - Browser: Safari

**Additional context**
Tested this on a few devices, but mostly on those ones. I did try to edit the vue myself but no luck so far. 

For the title bar, you can modify the PWA theme colour using the theme-color meta tag:
`<meta name="theme-color" content="#242629">`

**Comment by vialra** [comment]
I dont think this is an error the red color comes from GMS and is resolved for each main manufacturer (ssc is yellow for example)

**Comment by msprijatelj** [comment]
The Mech Page colors are intended; the PWA bar is probably not. So if there’s a way to change the PWA bar color, this issue can stay open for sure.

**Comment by vialra** [comment]
Oh you are totally right must have fumbled the PWA part while thinking about it :§ sorry.

**Comment by eth0net** [comment]
Ah okay I can understand the mech loadout page being their colours (although I'd personally prefer an option to control this behaviour :D)

But yeah the main window bar is fixed to red, regardless of theme or which part of the application you're currently in :)
